cypress-setup.sh

```diff
@@ -25,10 +25,11 @@ fi
 rm -rf e2e-projects/cypress-next-app \
 && npx --yes create-next-app e2e-projects/cypress-next-app \
 && npx --yes vite-node ./cypress/plugins/addAuth.ts -- ${EMAIL} ${PASSWORD} ${PRISMIC_URL} \
-&& npx --yes vite-node ./cypress/plugins/createRepo.ts -- "${_PRISMIC_REPO}" "${PASSWORD}" "${PRISMIC_URL}"  \
+&& npx --yes vite-node ./cypress/plugins/createRepo.ts -- "${_PRISMIC_REPO}" "${PASSWORD}" "${PRISMIC_URL}" \
 && npm --workspaces pack --pack-destination ./e2e-projects/cypress-next-app \
 && cd e2e-projects/cypress-next-app \
 && npm i *.tgz \
 && npx @slicemachine/init --repository ${_PRISMIC_REPO} \
+&& npm i --save-dev slice-machine-ui*.tgz \
```

Comment:
I added this because `npx @slicemachine/init` puts the `slice-machine-ui` production version dependency. I'm not sure if it's the best option, @MarcMcIntosh suggested to use Verdaccio for this but it can be done in a separate task.

Reply:
Works for now indeed